Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -654,12 +705,37 @@ void Wiimote::LoadDefaults(const ControllerInterface& ciface)
   m_dpad->SetControlExpression(3, "Right");  // Right
 #endif
 
+  // Motion Source
+  SetMotionSource(MotionSource::Emulated);
+  m_imu_accelerometer->SetControlExpression(0, "Accelerometer Left");
+  m_imu_accelerometer->SetControlExpression(1, "Accelerometer Right");
+  m_imu_accelerometer->SetControlExpression(2, "Accelerometer Backward");
+  m_imu_accelerometer->SetControlExpression(3, "Accelerometer Forward");
+  m_imu_accelerometer->SetControlExpression(4, "Accelerometer Up");
+  m_imu_accelerometer->SetControlExpression(5, "Accelerometer Down");
+  m_imu_gyro->SetControlExpression(0, "Gyro Pitch Up");
+  m_imu_gyro->SetControlExpression(1, "Gyro Pitch Down");
+  m_imu_gyro->SetControlExpression(2, "Gyro Roll Right");
+  m_imu_gyro->SetControlExpression(3, "Gyro Roll Left");
+  m_imu_gyro->SetControlExpression(4, "Gyro Yaw Right");
+  m_imu_gyro->SetControlExpression(5, "Gyro Yaw Left");
```

Comment:
A minor nitpick. All other mappings list "Left" before "Right" and "Forward" before "Backward". I'd swap some of these to keep that consistency. You'll have to reverse some subtractions elsewhere, of course.